WordPress/src/main/java/org/wordpress/android/ui/sitecreation/misc/SiteCreationTracker.kt

```diff
@@ -93,8 +93,12 @@ class SiteCreationTracker @Inject constructor(val tracker: AnalyticsTrackerWrapp
         tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SUCCESS_PREVIEW_LOADED)
     }
 
-    fun trackCreationCompleted() {
-        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_COMPLETED)
+    fun trackPreviewOkButtonTapped() {
+        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_PREVIEW_OK_BUTTON_TAPPED)
```

Comment:
Should we update the document tracking our event names?

Reply:
And open a ticket on the iOS side to rename [this](https://github.com/wordpress-mobile/WordPress-iOS/search?q=ENHANCED_SITE_CREATION_COMPLETED&unscoped_q=ENHANCED_SITE_CREATION_COMPLETED)

Reply:
iOS Issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/11809
iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/11829

Reply:
>Should we update the document tracking our event names?

Yes, I always update the document after the PR is merged. However, it makes sense to do it before the PR so the reviewer can review it as well! 

Updated!

Reply:
>And open a ticket on the iOS side to rename this

I thought updating it myself would be confusing as Diego wouldn't have the background and might not even notice the description changed so I just pinged him. Or do you think it's better to update it directly?

Thanks Diego for updating the iOS repo ;)